pages/glossary/standard-keyboard-navigation.md

```diff
@@ -3,6 +3,8 @@ title: Standard keyboard navigation
 key: standard-keyboard-navigation
 unambiguous: true
 objective: true
+input_aspects:
+  - DOM tree
```

Comment:
I'm not sure how this requires DOM. Might not have any input.
```suggestion
```

Reply:
It needs to know if elements are focusable, so it should include at least the same input aspects.